pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -224,6 +224,17 @@ let
       UCLAMP_TASK_GROUP = yes;
     };
 
+    # All of these need to be enabled to support sched-ext
+    sched-ext = {
+      BPF = whenAtLeast "6.12" yes;
+      BPF_SYSCALL = yes;
+      SCHED_CLASS_EXT = whenAtLeast "6.12" yes;
+      BPF_JIT            = whenPlatformHasEBPFJit yes;
+      BPF_JIT_ALWAYS_ON  = whenPlatformHasEBPFJit (whenAtLeast "6.12" yes); # whenPlatformHasEBPFJit yes; # see https://github.com/NixOS/nixpkgs/issues/79304
+      BPF_JIT_DEFAULT_ON = whenPlatformHasEBPFJit yes;
+      PAHOLE_HAS_SPLIT_BTF = whenAtLeast "6.12" yes;
```

Comment:
These should be autodetected from pahole (and in fact already are on our existing kernels).